Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/client/CONNECTCXFClientFactory.java

```diff
@@ -64,7 +69,10 @@
         if (StringUtils.isEmpty(url)) {
             throw new WebServiceException(ERROR);
         }
-
+        if (StringUtils.isBlank(target.getExchangeName())) {
```

Comment:
Im handling this logic already in `CONNECTCXFClientSecured`. I think we should just update the logic already there. Additionally the other getter here (without the NhinTargetSystemType) above on line 51 also needs to make use of this change.

See https://github.com/CONNECT-Solution/CONNECT/pull/1851/files#diff-c7700e56eac1a9d3585fc702a81e867eR79

Reply:
The logic here is different.  We want to handle the default exchange if empty since our NHIN layer only call this method.